docs/documentation/Realtime/recent additions.md

```diff
@@ -35,10 +20,36 @@ Join the discussions on <a href="https://github.com/google/transit/pulls" target
 </div>
 <div class="row"></div> -->
 
-## Recently Merged Proposals &ensp;<img src="../../assets/pr-merged.svg" style="height:1em;"/>
+## Recently Merged Proposals &ensp;<img src="../../../assets/pr-merged.svg" style="height:1em;"/>
+
+Recently merged proposals that are now features of the [official GTFS Realtime specification](../reference). See the complete Revision History for more.
+
 
-Recently merged proposals that are now features of the [official GTFS Realtime specification](../reference). See the complete [Revision History](../process#revision-history) for more.
```

Comment:
Since we removed the revision history from the RT Spec Amendment process we don't have anywhere to refer for this information. From what I can see there is no "official" revision history for RT in the google/transit repo. Any thoughts on how can we solve this? @isabelle-dr @tzujenchanmbd

Reply:
Interesting; the revision history for Realtime only exists on the gtfs.org repo?

From a user experience on gtfs.org perspective, I think the revision history belongs to the same place where this page is (I think the technical documentation is a good place to have this). 
The revision history serves the following user stories:
- I what to know what were the GTFS additions in the past three years
- I want to know when continuous stops was adopted in GTFS
The "Recent Additions" page is something we added to answer 90% of what folks are looking for in a revision history in half the time because there are not all the details no one cares about.

IMO it goes either after this table on this page, or in another one. If it was in another one, we'd have something like:
- documentation/change history
- documentation/change history/recently adopted -> this page
- documentation/change history/complete change history -> the full revision history

Reply:
Maybe both need to be their separate file in google/transit also